libnavigation-core/src/main/java/com/mapbox/navigation/core/lifecycle/MapboxNavigationOwner.kt

```diff
@@ -60,6 +60,7 @@ internal class MapboxNavigationOwner {
             attached = false
             services.forEach { it.onDetached(mapboxNavigation!!) }
             MapboxNavigationProvider.destroy()
+            logI("disabled ${services.size} observers", LOG_CATEGORY)
```

Comment:
At the time this is disabled, there are 11 services attached because of this

https://github.com/mapbox/mapbox-navigation-android/blob/156c544c22c1b911c6bb7e7c9b3aae99107b3221/qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/QaTestApplication.kt#L11

It hooks up to the `SharedApp`

Reply:
Can consider that a separate issue. Adding this log is helpful to keep track of observers that should not be registered.

Reply:
Tracking issue here https://github.com/mapbox/mapbox-navigation-android/issues/6253